src/lib/error-rate-check.js

```diff
@@ -0,0 +1,29 @@
+const nHealth = require('n-health');
+
+const DEFAULT_THRESHOLD = 0.04;
+const DEFAULT_SEVERITY = 1;
```

Comment:
on reflection, at least while trialling this, we should probably set it to low severity